modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/ConvertProcessor.java

```diff
@@ -81,10 +99,16 @@ public Object convert(Object value) {
                 }
                 try {
                     return BOOLEAN.convert(value);
-                } catch (IllegalArgumentException e) { }
+                } catch (IllegalArgumentException e) {}
                 try {
                     return INTEGER.convert(value);
                 } catch (IllegalArgumentException e) {}
+                try {
+                    return LONG.convert(value);
+                } catch (IllegalArgumentException e) {}
+                try {
+                    return DOUBLE.convert(value);
```

Comment:
move the double parsing below the float parsing? So that it is consistent with integer/long auto parsing.

Reply:
sure, committed :)